### ISSUE TITLE
cant login cpr 

### ISSUE THREAD

**Issue by anoyn231** [relrelb/CPPS-PCL#22]
flash is unable to be enabled and keeps sending me to the download flash site when the link is clicked. cpr works on chrome and also doesn't work with cprflashenabler plugin installed. 

this also get displayed;

> CPPS: cpr                                                                               
Server: blizzard                                                                                     
Shape: square                                                                                                       
Logging in with 4 penguins...       
                                                                                                                                                                                                           DevTools listening on ws://127.0.0.1:53048/devtools/browser/9584e7db-a5a3-4775-a32c-1d2d185faa48                        [0706/092824.612:ERROR:cef_log.cpp(16)] [Browser process] SetBounds() not implemented on this platform    

> 


![Screenshot_418](https://user-images.githubusercontent.com/52602093/60756826-78c98f00-9fd0-11e9-9fde-37b5d461aea8.png)

**Comment by anoyn231** [comment]
this might be useful; the cefpython3 version 66.0

> C:\Users\CMD.DESKTOP-S7KT2VR>python
Python 2.7.16 (v2.7.16:413a49145e, Mar  4 2019, 01:37:19) [MSC v.1500 64 bit (AMD64)] on win32                          Type "help", "copyright", "credits" or "license" for more information.
import cefpython3              
print cefpython3.__version__   
66.0          

>

**Comment by anoyn231** [comment]
possibly the chromium needs a flash plugin? not really sure

**Comment by relrelb** [comment]
Hello anoyn231, thank you for providing the cefpython version.

As for the reCAPTCHA window showing CPR page, it has nothing to do with flash at all. In order to display the reCAPTCHA window, I had to load CPR webpage, then detect that request and replace its content with `reCAPTCHA.html`. Probably due to changes in cefpython, the script could not detect the CPR webpage request and therefore could not change it to `reCAPTCHA.html`. To fix that issue, I loaded CPR webpage after setting up the client handler, so it must detect it.

As for the `SetBounds` issue, cefpython has not yet implemented it to Windows, and it works only on Linux platforms.
